### PR TITLE
add document for include-conf feature

### PR DESCRIPTION
**Pull request by i110**
@kazuho Added a document for https://github.com/h2o/h2o/pull/1022

### CODE REVIEW

File: srcdoc/configure/syntax_and_structure.mt [hunk 1]
```diff
@@ -125,4 +125,39 @@ hosts:
 EOT
 ?>
 
+<h3 id="include_other_files">Using Inclusion</h3>
```
Comment (kazuho) [inline srcdoc/configure/syntax_and_structure.mt]: Change to `Including Files`?

Sorry if my suggestion to add a _using_ section has led to this choice of wording.

File: srcdoc/configure/syntax_and_structure.mt [hunk 2]
```diff
@@ -125,4 +125,39 @@ hosts:
...
+
+<p>
+Since version 2.1, H2O makes it possible to include other config files using <code>!file</code> custom YAML tag.
```
Comment (kazuho) [inline srcdoc/configure/syntax_and_structure.mt]: s/Since/Starting from/ might be better considering that we are going to release 2.1.

Comment (kazuho) [inline srcdoc/configure/syntax_and_structure.mt]: s/H2O makes it possible to include other config files/it is possible to include a YAML file from the configuration file/ ?

**Comment by i110** [comment]
@kazuho thank you for your comments. I addressed the points you mentioned.

**Pull request merged**